tmt/utils.py

```diff
@@ -3265,7 +3265,8 @@ def _patch_plan_schema(schema: Schema, store: SchemaStore) -> None:
         step_schema_prefix = f'/schemas/{step}/'
 
         step_plugin_schema_ids = [schema_id for schema_id in store.keys() if schema_id.startswith(
-            step_schema_prefix) and schema_id != '/schemas/provision/hardware']
+            step_schema_prefix) and schema_id != '/schemas/provision/hardware'
+            and schema_id != '/schemas/provision/kickstart']
```

Comment:
Yeah, once it's two or more, it's no longer an exception :) Could you take these two files & make a global list out of them? Something like `PLAN_SCHEMA_IGNORED_IDS: List[str] = [...]`, and use it here to skip ids we don't want to inject into plan.

Reply:
Make sense, I'll add it